vllm/envs.py

```diff
@@ -961,7 +961,11 @@ def get_vllm_port() -> Optional[int]:
     # consumer. This is only applicable when using NixlConnector in a
     # disaggregated decode-prefill setup.
     "VLLM_NIXL_ABORT_REQUEST_TIMEOUT":
-    lambda: int(os.getenv("VLLM_NIXL_ABORT_REQUEST_TIMEOUT", "120"))
+    lambda: int(os.getenv("VLLM_NIXL_ABORT_REQUEST_TIMEOUT", "120")),
+
+    # If set to 1, use the TRTLLM Decode Attention backend in flashinfer.
+    "VLLM_USE_TRTLLM_DECODE_ATTENTION":
+    lambda: os.getenv("VLLM_USE_TRTLLM_DECODE_ATTENTION", None),
```

Comment:
I think we should convert it to a bool like other envs, then it will make the checks cleaner

```suggestion
    lambda: bool(int(os.getenv("VLLM_USE_TRTLLM_DECODE_ATTENTION", "0")))
```

Reply:
To enable switching without specifying the environment variable, the default value must not be set. So using None here seemed like the better choice. Please let me know I can do it differently.

Reply:
Oh I didn't know about this limitation, got it